tests/test.cc

```diff
@@ -1,2 +1,30 @@
+#include "node.h"
+
 #define CATCH_CONFIG_MAIN
 #include "catch.hpp"
+
+TEST_CASE( "Node is checked", "[Node]" ) {
+	Node test (123, {11.2, 22.3, 33.4});
+
+	REQUIRE( test.id() == 123 );
```

Comment:
Always use variables, random numbers don't mean anything. I don't know why it should be 123.